lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -436,28 +428,41 @@ func convertContainerToContainerInfo(container *exec.ContainerInfo) *models.Cont
 		}
 	}
 
-	path := container.ExecConfig.Sessions[ccid].Cmd.Path
-	info.ProcessConfig.ExecPath = &path
+	// in heavily loaded environments we were seeing a panic due to a missing
+	// session id in execConfig -- this has only manifested itself in short lived containers
+	// that were initilized via run
```

Comment:
No - I don't know the exact cause of the issue -- to your point I have only avoided the panic.  This happens in the CI intermittently and when the environment is apparently loaded / slow -- the test failed due to the panic, but would have failed because of network issues.

We can leave this change out and leave the issue open -- but reproduction is a challenge.  I was banking on the avoidance of a panic being the preferred approach.